janusgraph-examples/example-remotegraph/conf/remote-objects.yaml

```diff
@@ -0,0 +1,8 @@
+hosts: [127.0.0.1]
+port: 8182
+serializer: {
```

Comment:
This is mixing JSON in a YAML config, no? I understand that JSON is a subset of YAML, but this can also be written as:

```yaml
serializer:
  className: org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV1d0
    config:
      ioRegistries: [org.janusgraph.graphdb.tinkerpop.JanusGraphIoRegistry]
```

Or would that not do the same thing?

Reply:
Using curly braces is the same style used in the reference TinkerPop [remote-objects.yaml](https://github.com/apache/tinkerpop/blob/3.2.6/gremlin-console/conf/remote-objects.yaml).

Reply:
And it appears that the Java parser wants those curly braces in place otherwise it fails to parse the flow mapping.

Reply:
I am in favor of keeping in sync with TinkerPop style. However, for education purposes only, the correct pure YAML format is (and works with TinkerPop):
```
hosts: 
  - localhost
port: 8182
serializer: 
  className: org.apache.tinkerpop.gremlin.driver.ser.GryoMessageSerializerV1d0
  config: 
    ioRegistries: 
      - org.apache.tinkerpop.gremlin.tinkergraph.structure.TinkerIoRegistryV1d0
```

Reply:
LGTM